lib/pbench/agent/results.py

```diff
@@ -178,12 +187,12 @@ def copy_result_tb(self):
                 file_count,
                 self.tarball,
             )
-            sys.exit(1)
+            return 1
 
         with open(self.tarball_md5, "r") as md5fp:
             md5sum = md5fp.read().split()[0]
         filename = secure_filename(str(self.tarball))
-        headers = {"filename": filename, "Content-MD5": md5sum}
+        headers = {"filename": filename, "Content-MD5": md5sum, "token": token}
```

Comment:
The authorization token needs to be presented as `"Authorization": f"Bearer {token}"`

Reply:
sure